public/CSS/style.css

```
@@ -0,0 +1,161 @@
+/** reset **/
+* {
+    margin: 0;
+    padding: 0;
+  }
+
+  a {
+    text-decoration: none;
+    color: inherit;
+  }
+  
+  /* True Styling */
+  
+  header {
+    background-image: url(https://actualitte.com/media/cache/width_944_webp/uploads/images/un-nouveau-projet-d-hayao-miyazaki-inspire-d-un-classique-de-la-litterature-japonaise-6950fb53c799.jpg);
+    background-size: cover; 
+    background-repeat: no-repeat;
+    display: flex;
+    flex-direction: column;
+  }
+  
+  header h1 {
+    margin: 0;
+    font-family: fantasy;
+    font-size: 5rem;
+    padding: 10px;
+    text-align: center;
+    color: white!important;
```

```suggestion
    color: white;
```

Eviter au maximum le `!important` : si tu trouves une autre solution, ça serait idéal. Sinon ça ira :)